src/ripple/ledger/ApplyView.h

```diff
@@ -212,6 +213,67 @@ class ApplyView
         std::uint32_t cur, std::uint32_t next)
     {};
 
+    /** Add an entry to a directory
+
+     @param directory the base of the directory
+     @param key the entry to insert
+     @param strictOrder keep entries in order of insertion
```

Comment:
I like this.  Perhaps a better name for `dirInsertUnsorted` would be `dirAppend`?